JournalCoreData/Model Controllers/EntryController.swift

```diff
@@ -129,6 +133,7 @@ class EntryController {
                     completion(error)
                 }
             }
+            completion(nil)
```

Comment:
This might be redundant? We would need to run the code to find out. See line 130